readme.md

```diff
@@ -124,6 +124,7 @@ Click the type names for complete docs.
 - [`Includes`](source/includes.d.ts) - Returns a boolean for whether the given array includes the given item.
 - [`Simplify`](source/simplify.d.ts) - Flatten the type output to improve type hints shown in editors.
 - [`Jsonify`](source/jsonify.d.ts) - Transform a type to one that is assignable to the `JsonValue` type.
+- [`Exactly`](source/exactly.d.ts) - Create a guard for function to reject excess properties.
```

Comment:
The type is not just about guarding functions.

Reply:
I can only think of guarding a function because it requires input type from a function input. Could you suggest other usages?

Reply:
The descriptions describes *one* of the uses of the type, but the description should describe what it does.